### ISSUE TITLE
Mother Slimes Instabreak Shields and Hitboxes all around are really unbalanced.

### ISSUE THREAD

**Issue by HisashiASM** [lucaargolo/terrarian-slimes#13]
Tested on 1.18.1 via the Better Minecraft mod.
The slimes have too big of a Hitbox IMO, I cannot deal with them in Hardcore Survival because I am not even allowed to approach them.
Mother Slime legit just deletes shields, and may do too much damage, seems unreasonable to do like 8.5 heart damage with gold gear.
This is a mod I really wanna enjoy, but it is impossible to use with Hardcore.

**Comment by milkev** [comment]
i agree with the hitboxes. they seem to be a similar size to hoglins, but on a mob thats far more common and spawns everywhere. It would probably be better if the hitboxes were just the default minecraft slime hitbox size
